docs/developer/typescript/01_gettingStarted/01_hello/02_modules.md

```diff
@@ -0,0 +1,8 @@
+# Module Imports
+```ts title="ee-tutorial-hello/src/Hello.ts"
+import { ECS } from '@etherealengine/ecs'
+import { NameComponent } from '@etherealengine/spatial/src/common/NameComponent'
+import { VisibleComponent } from '@etherealengine/spatial/src/renderer/components/VisibleComponent'
+import { TransformComponent } from '@etherealengine/spatial/src/transform/components/TransformComponent'
+import { PrimitiveGeometryComponent } from '@etherealengine/engine/src/scene/components/PrimitiveGeometryComponent'
```

Comment:
I feel like the tutorial should be simple enough that it is on one page. So that it's a good refrence. If it's simple enough it shouldn't be overwhelming.

Reply:
It might be. But I haven't reached this part of the guide yet, so I haven't had the time to confirm if this is the case. So kept it as a separate file for now to order my thoughts.

Reply:
As a sidenote, this structure is trying to mirror the structure of:
https://tutorial.docusaurus.io/docs/category/tutorial---basics
Which could also be argued that *should* fit one page because its simple... but it's actually much more digestible being separate pages.
That's the reasoning behind it, but as mention will double check when the writing reaches this page.

Reply:
The main issue is that users do not want to click. It's a very common UX law. Limit the amount a user has to click to get something done. If we're going over an example file it should all be done in one space. The context is the same. We create a new page for a completely new context. If contexts are related we should try our best to keep them together if possible. For example, keyboard and gamepad inputs.

Reply:
https://github.com/EtherealEngine/etherealengine-docs/commit/4b12554c462bf23e95e44272cf8b5d7f9d433666